app.ts

```diff
@@ -0,0 +1,137 @@
+let recordCount: number, startIndex: number, endIndex: number;
+let columnHeaders: any[];
+let resizeTimer: number , rowsPerView : number;
```

Comment:
Formatting is a bit off here. You might want to check that your vscode is formatting your code.